content/en/research/_index.md

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Research documentation",
+  "weight": 0,
+  "bookCollapseSection": true,
+  "bookFlatSection": true,
+  "cascade": [
+        {
+            "type": "research",
+            "_target": {
+                "path": "/research/**"
+            }
+        }
+    ]
+}
+
+# Research documentation
+
+In the documentation for the tracker adapters, we attribute types of data to paths in the decoded requests. These attributions are the result of our research and we provide the reasons in the tables in the tracker documentation and in the adapter objects in TrackHAR (in the `reasoning` field), so that others don’t have to trust our results blindly. We decided on a finite set of justification that we will give listed here in the order of importance that we will go through when choosing a reason:
```

Comment:
Also, I don't think 'importance' fits here.

Reply:
What do you think would fit better? "persuasiveness"? Or just the second part behind the "that"?

Reply:
> Or just the second part behind the "that"?

Yeah, that's good.